client/lib/abtest/active-tests.js

```diff
@@ -133,4 +133,13 @@ export default {
 		defaultVariation: 'control',
 		allowExistingUsers: true,
 	},
+	domainShowJPResultsInJapan: {
```

Comment:
You did not define a `defaultVariation` and it's throwing an error in the console:

```
Error: No default variation found for domainShowJPResultsInJapan
```

Reply:
Sorry -- I merged/rebased and missed this during conflict resolution. 🤦